featuretests/cmd_juju_status_test.go

```diff
@@ -20,9 +21,7 @@ type StatusSuite struct {
 	jujutesting.JujuConnSuite
 }
 
-func (s *StatusSuite) SetUpTest(c *gc.C) {
-	s.JujuConnSuite.SetUpTest(c)
-
+func (s *StatusSuite) assertMultipleRelationsBetweenApplications(c *gc.C) {
```

Comment:
Maybe setupMultipleRelationsBetweenApplications
assertFoo() is typically used to test the results of doing something after it has happened and we want to check that the code behaved as expected.
Here, we are setting up a scenario for which we then want to run a test